GVRf/Framework/jni/shaders/material/assimp_shader.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/***************************************************************************
+ * Shader for model loaded with Assimp
+ ***************************************************************************/
+
+#ifndef ASSIMP_SHADER_H_
+#define ASSIMP_SHADER_H_
+
+#include <memory>
+
+#include "GLES3/gl3.h"
+#include "glm/glm.hpp"
+#include "glm/gtc/type_ptr.hpp"
+
+#include "objects/recyclable_object.h"
+
+#define SETBIT(num, i)                   num = (num | (1 << i))
+#define ISSET(num, i)                    ((num & (1 << i)) != 0)
+#define CLEARBIT(num, i)                 num = (num & ~(1 << i))
+
```

Comment:
Just realized this seems a bug. Used some old code, so didn't paid much attention.